langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModel.java

```diff
@@ -69,12 +71,30 @@ public void setDetails(OllamaModelDetails details) {
         this.details = details;
     }
 
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public OffsetDateTime getModifiedAt() {
+        return modifiedAt;
+    }
+
+    public void setModifiedAt(OffsetDateTime modifiedAt) {
+        this.modifiedAt = modifiedAt;
+    }
+
     static class Builder {
 
         private String name;
         private long size;
         private String digest;
         private OllamaModelDetails details;
+        private OffsetDateTime modifiedAt;
+        private String model;
 
         Builder name(String name) {
```

Comment:
Should all builder methods also be public?

Reply:
Same answer as above